cloud/aws/variables.tf

```diff
@@ -6,6 +6,11 @@ variable "cockroach_port" {
   default = "26257"
 }
 
+# Comma-separated list of directories added as auxiliary stores.
+variable "aux_data_dirs" {
+  default = ""
```

Comment:
yeah, that's weird. I would make the default `"data"`, and keep it out of the `formatlist`